sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -351,9 +312,16 @@ function buildAuthorizationRule(value: any): AuthorizationRule {
  * @param value
  */
 export function getRawAuthorizationRules(authorizationRules: AuthorizationRule[] | undefined): any {
-  if (!Array.isArray(authorizationRules)) {
+  if (authorizationRules == undefined) {
     return undefined;
   }
+
+  if (!Array.isArray(authorizationRules)) {
+    throw new TypeError(
+      `authorizationRules must be an array of SqlParameter instances or undefined`
```

Comment:
Since this error is for user provided input, we should consider including the given input in the error message.
@bterlson thoughts?

Reply:
Updated

Reply:
Generally speaking: omit the input value when you can be clear to the user what the error is talking about by referring to e.g. a parameter or option name. However, this is a case where we'll fall over on a part of a giant object and authorization rules is also an array, so just json stringifying the object seems like the best way to tell a user which authorizationRules entry is malformed.